kit/azure/bootstrap/outputs.tf

```diff
@@ -1,8 +1,3 @@
-output "client_id" {
-  value = azuread_service_principal.cloudfoundation_deploy.application_id
-}
-
-output "client_secret" {
-  value     = azuread_service_principal_password.cloudfoundation_deploy.value
-  sensitive = true
-}
+output "terraform_state" {
```

Comment:
d: we write this to a file anyway, right? why do we put it into tf output as well? if it's not used anywhere, we should probably drop it